README.md

```diff
@@ -36,14 +36,21 @@ To view a live example, **[click here](https://saadpasta.github.io/)**
 
 These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-You'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer
+You'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer or use [Docker](https://www.docker.com/products/docker-desktop).
 
 ```
 node@v10.16.0 or higher
 npm@6.9.0 or higher
 git@2.17.1 or higher
 ```
 
+### Docker Commands 
+
+```
+1) BUILD IMAGE : docker build -t developerfolio:latest .
+2) RUN IMAGE: docker run -p 3000:3000 developerfolio:latest
+```
+
 ---
```

Comment:
@saadpasta do you think this breaker needs to exist since it looks pretty inconsistent with the rest of the documentation?

Reply:
Also move these changes in a single commit under your already existing PR if you can @viveksharmaui

Reply:
@kartik918 sure!

Reply:
@kartik918 What do you say ?

Reply:
Should we move this at the end

Reply:
Yes, that would be better. @viveksharmaui move the hr breaker to the end and update docker instructions in readme in your existing "Dockerized" branch PR. I'll close this PR after that one gets merged.